modules/serverless-channels-creating-intro.adoc

```diff
@@ -30,7 +30,6 @@ The `spec.channelTemplate` properties cannot be changed after creation, because
 
 The channel controller then creates the backing channel instance based on the `spec.channelTemplate` configuration.
 
-When this mechanism is used with the example above, two objects are created: a generic backing channel and an `InMemoryChannel` channel.
-If you are using a different default channel implementation, for example, Apache Kafka, a generic backing channel and `KafkaChannel` channel are created.
+When this mechanism is used with the example above, two objects are created: a generic backing channel and an `InMemoryChannel` channel. If you are using a different default channel implementation, for example, Knative Kafka, a generic backing channel and `KafkaChannel` channel are created.
```

Comment:
> ! above _adjective, adverb, preposition_
... Do not use to indicate a relative location in a document, as in "the above restrictions" or "the steps above". Use "previous" or "preceding". Examples: "The previous restrictions apply to ..."; "The preceding steps..."

This makes it sound like if you're using any other channel implementation, you get the `KafkaChannel`. Do you mean something like:

```suggestion
When this mechanism is used with the preceding example, two objects are created: a generic backing channel and an `InMemoryChannel` channel. If you are using a different default channel implementation,  the `InMemoryChannel` is replaced with one that is specific to your implementation. For example, with Knative Kafka, the `KafkaChannel` channel is created.
```

Reply:
Perfect, thank you